deb/build

```diff
@@ -22,6 +22,38 @@ genchangelog()
     echo " -- $3  `LANG=C date -R`"
 }
 
+versionedlibrary()
+{
+    # package major version
+    pkgsuffix=$1
+
+    curr_dir=$PWD
+
+    # cd to the `make` install path
+    cd $2
+
+    # rename the original library file to the versioned library file
+    mv $pkgname.so.$libversion "$pkgname$pkgsuffix.so.$libversion"
+    mv $pkgname.a "$pkgname$pkgsuffix.a"
+    mv $pkgname.la "$pkgname$pkgsuffix.la"
+
+    # remove possibly invalid symbolic links
+    find . -type l -name "$pkgname.*" -exec rm '{}' \;
+
+    # set `pkgname` to the versioned package name
+    pkgname=$pkgname$pkgsuffix
+
+    # the versioned library name
+    so_libname=$pkgname.so
+
+    # create symbolic links for the renamed library
+    ln -sf $so_libname.$libversion $so_libname.$lib_major
+    ln -sf $so_libname.$lib_major $so_libname
+
+    cd $curr_dir
```

Comment:
No need to save this, you can access the previous directory with `$OLDPWD` and you can just use `cd -` as a shortcut to `cd $OLDPWD`.